File: Milk/Beer.java
class Main {
    public static void main(String[] args) {
        for (String arg : args) System.out.println(arg);
        for (int i = 99; i > 0; i--) {
            System.out.println(i + " bottle" + (i > 1 ? "s" : "") + " of beer on the wall, " + i + " bottle" + (i > 1 ? "s" : "") + " of beer.");
            System.out.println("Take one down and pass it around, " + (i - 1 > 0 ? (i - 1) + " bottle" + (i - 1 > 1 ? "s" : "") : "no more bottles") + " of beer on the wall.\n");
        }
        System.out.println("No more bottles of beer on the wall, no more bottles of beer.");
        System.out.println("Go to the store and buy some more, 99 bottles of beer on the wall.");
    }
}

File: Milk/Beer.rs
fn main() {
    let mut bottles = 99;

    while bottles > 0 {
        println!(
            "{} bottle{} of beer on the wall, {} bottle{} of beer.",
            bottles,
            if bottles == 1 { "" } else { "s" },
            bottles,
            if bottles == 1 { "" } else { "s" }
        );
        bottles -= 1;
        println!(
            "Take one down and pass it around, {} bottle{} of beer on the wall.\n",
            if bottles == 0 { "no more".to_string() } else { bottles.to_string() },
            if bottles == 1 { "" } else { "s" }
        );
    }

    println!("No more bottles of beer on the wall, no more bottles of beer.");
    println!("Go to the store and buy some more, 99 bottles of beer on the wall.");
}